distribution/archives/integ-test-zip/src/test/java/org/elasticsearch/test/rest/NodeRestUsageIT.java

```diff
@@ -80,24 +81,28 @@ public void testWithRestUsage() throws IOException {
         }
 
         // Do some requests to get some rest usage stats
-        client().performRequest("PUT", "/test");
-        client().performRequest("POST", "/test/doc/1", Collections.emptyMap(),
-                new StringEntity("{ \"foo\": \"bar\"}", ContentType.APPLICATION_JSON));
-        client().performRequest("POST", "/test/doc/2", Collections.emptyMap(),
-                new StringEntity("{ \"foo\": \"bar\"}", ContentType.APPLICATION_JSON));
-        client().performRequest("POST", "/test/doc/3", Collections.emptyMap(),
-                new StringEntity("{ \"foo\": \"bar\"}", ContentType.APPLICATION_JSON));
-        client().performRequest("GET", "/test/_search");
-        client().performRequest("POST", "/test/doc/4", Collections.emptyMap(),
-                new StringEntity("{ \"foo\": \"bar\"}", ContentType.APPLICATION_JSON));
-        client().performRequest("POST", "/test/_refresh");
-        client().performRequest("GET", "/_cat/indices");
-        client().performRequest("GET", "/_nodes");
-        client().performRequest("GET", "/test/_search");
-        client().performRequest("GET", "/_nodes/stats");
-        client().performRequest("DELETE", "/test");
+        client().performRequest(new Request("PUT", "/test"));
+        Request request = new Request("POST", "/test/doc/1");
+        request.setJsonEntity("{\"foo\": \"bar\"}");
+        client().performRequest(request);
+        request = new Request("POST", "/test/doc/2");
+        request.setJsonEntity("{\"foo\": \"bar\"}");
```

Comment:
I really dislike this style of variable reuse in our tests. If I use my IDE to navigate to the definition of this variable I end up on a line assigning a value to this variable that is removed from its current value. This hinders readability, especially in longer tests. Let’s avoid introducing it here, we should be moving away from it.

Reply:
Do you prefer extra blocks or different names to stop the reuse?